src/clj/game/cards/operations.clj

```diff
@@ -291,6 +291,11 @@
                               :effect (effect (trash target))}
                             card nil)))}}
 
+   "Enhanced Login Protocol"
+   {:events {:pre-click-run {:once :per-turn
+                             :msg "increase the cost of running"
+                             :effect (effect (click-run-bonus [:click 1]))}}}
```

Comment:
It might be worth taking a look at how #2071 implemented Service Outage (though obviously only add additional costs to click runs). I made Service Outage responsible for removing the bonus it added.